### PR TITLE
Simplify lock file

### PR DESCRIPTION
**Pull request by turion**
I removed some duplicate inputs.

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -13,15 +13,27 @@
...
+    pre-commit-hooks = {
+      url = "github:cachix/pre-commit-hooks.nix";
+      inputs = {
+        nixpkgs.follows = "nixpkgs";
+        flake-utils.follows = "flake-utils";
+      };
+    };
     haskell-nix-utils.url = "github:TerrorJack/haskell-nix-utils";
-    jupyterWith.url = "github:tweag/jupyterWith";
-    # jupyterWith.inputs.nixpkgs.follows = "nixpkgs";
```
Comment (turion) [inline flake.nix]: @idontgetoutmuch do you remember why this was commented? Is there harm in keeping this in sync with our nixpkgs input?

**Comment by turion** [comment]
If we merge this, we can close #207.

File: flake.nix (hunk 1 above)
Comment (turion) [reply flake.nix]: Ah, I guess because it simply doesn't build like that.

Comment (turion) [reply flake.nix]: This is probably https://github.com/tweag/monad-bayes/pull/231

Comment (turion) [reply flake.nix]: Let's see whether it works after a rebase.

**Comment by turion** [comment]
This PR introduces a lot of formatting changes which may make rebasing harder.
To rebase after this is merged, one should probably do the following:

* Rebase onto a commit directly before this, manually resolving further conflicts
  * `git rebase e1408795f3923c4eff81373d9e6fce47632bc038`
* Rebase onto the ormolu commit from this PR, automatically resolving conflicts and applying formatting
  * `git rebase 9605aa02a1b92b6d5d725db3129d74a60d45798f -X theirs --exec 'ormolu -i $(git ls-files "*.hs"); git commit -a --amend --no-edit'`
* Rebase onto master
  * `git rebase origin/master`

**Review by idontgetoutmuch — APPROVED** [review]

**Pull request merged**